tpot/base.py

```diff
@@ -751,6 +754,31 @@ def set_params(self, **params):
 
         return self
 
+    def clean_pipeline_string(self, individual):
+        """
```

Comment:
Minor issue: Please remove the newline between the `"""` and the start of the docstring text.